@navikt/core/react/src/date/Time/Time.tsx

```diff
@@ -0,0 +1,65 @@
+import { format } from "date-fns";
+import { nb } from "date-fns/locale";
+import React from "react";
+
+type Props = {
+  /**
+   * The date you want to format in accordance
+   * with our guidelines for writing dates and times.
+   *
+   * https://aksel.nav.no/god-praksis/artikler/skriveregler-i-nav
+   */
+  date: Date;
+} & MutexProps;
+
+type MutexProps =
+  | {
+      day?: boolean;
+      month?: never;
+      onlyTime?: never;
+    }
+  | {
+      day?: never;
+      month?: boolean;
+      onlyTime?: never;
+    }
+  | {
+      day?: never;
+      month?: never;
+      onlyTime?: boolean;
+    };
+
+/**
+ * A component that displays a point in time in accordance with
+ * our best practice guidelines for displaying dates and times.
+ *
+ * https://aksel.nav.no/god-praksis/artikler/skriveregler-i-nav
+ *
+ * @see [📝 Documentation](https://aksel.nav.no/komponenter/core/time)
+ * @see 🏷 {@link Props}
+ *
+ * @example
+ * ```jsx
+ * <Time date={new Date()} />
+ * <Time date={new Date()} day />
+ * <Time date={new Date()} month />
+ * <Time date={new Date()} onlyTime />
+ * ```
+ */
+export const Time = ({
+  date,
+  day = false,
+  month = false,
+  onlyTime = false,
+}: Props) => {
+  let formatString = "d. MMMM yyyy 'kl'. hh.mm";
```

Comment:
The space between `d.` and `MMMM` should be non-breaking.

Reply:
So now a new idea (after discussion with Ken) is to not have these as render components, but maybe serve some small utility library of functions. 🤔 This way we don't have issues with breaking spaces / any render issues. Though I do like the `<span>` margin padding for phone numbers 🤔... What do you think about a small utility library?

immediate con:
- so far we only have components (and CSS / tokens / patterns on how to use components... These are fairly closely tied to components), and this would break that "simplicity".